hrm-domain/hrm-service/src/bin/khpDataPull.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import { pullData } from '../data-pull-task/khp-data-pull-task';
+
+/**
+ * Returns an object with the arguments passed to the script.
+ *
+ * Example:
+ * > npm run start:khp-data-pull -- --start-date=2023-05-01 --end-date=2023-05-30
+ *
+ * returns:
+ * { start-date: '2023-05-01', end-date: '2023-05-30' }
+ */
+// Question: Should use a lib for this?
```

Comment:
Do we even need to use named arguments? It doesn't feel like there will be more than 2 options to this and I can't think of a use case where anyone would want to provide an endDate without a start date. Am I missing an intended use case where ordered arguments wouldn't work?

Reply:
The arguments don't necessarily need to be named. But I've assumed it's common practice. It would be very easy to change, though.